### ISSUE TITLE
feat(nimble): nimble url link to nimble

### ISSUE THREAD

**Issue by hugosenari** [Yardanico/nimeventer#2]
**Actual Behavior**:
Nimble package URL point to project URL

_Example_:
> New Nimble package! config - A library for working with the CFG configuration format, see
> https://docs.red-dove.com/cfg/index.html

**Expected Behavior**:
Nimble package URL point to nimble package URL

_Example_:
> New Nimble package! config - A library for working with the CFG configuration format, see 
> https://nimble.directory/pkg/config

**Additional Info**:

Is feature request, please reject if don't make any sense.

**Why:**
- Cohesion, same behavior of #1
- Project URL may not be package project URL

**Why not:**
- For most cases user really wants project URL

**Comment by Yardanico** [comment]
Sure, I'll change it, it's just that almost everyone used the same link for web and url :)

**Comment by hugosenari** [comment]
>I honestly think that package developers should provide the correct link to the web URL so that no one gets confused.

Agree, but they can provide any affiliate link or crypto scam instead :see_no_evil: 

> Also do you think we really should provide the nimble.directory link instead of the normal URL field linking to GitHub or some other repo platform?

Since is a _nimble directory_ new package announcement, it makes sense link to _nimble directory_ and there they also have link to whatever developer puts in Web URL

But is bikeshedding, so you decide, anyway I'm happy with current behavior 99% of time. Thanks, really.

**Comment by Yardanico** [comment]
@hugosenari "Since is a nimble directory new package announcement, it makes sense link to nimble directory and there they also have link to whatever developer puts in Web URL" not sure what you mean by that :) nimble.directory is an semi-official (not officially supported by the Nim team) website. It's not directly affiliated with Nimble itself

**Comment by hugosenari** [comment]
I wasn't aware of that, in this case ignore that point.

UX perspective: It still has some value as single source of truth with almost same behavior no matter what package creator links to.
Could be another site if there are any, but would me much better if it was an official source.
